### PR TITLE
CTOR-1053 : Refresh cloud-aws-cloudwatchlogs

### PR DESCRIPTION
**Pull request by lucie-dubrunfaut**
## Description

Refresh cloud-aws-cloudwatchlogs

## Target version (i.e. version that this PR changes)

- [ ] 22.10.x
- [ ] 23.04.x
- [ ] 23.10.x
- [ ] 24.04.x
- [ ] 24.10.x
- [ ] Cloud
- [x] Monitoring Connectors

### CODE REVIEW

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md [hunk 1]
```diff
@@ -5,33 +5,52 @@ title: Amazon CloudWatch Logs
 import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
+## Dépendances du Connecteur de supervision
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md:8]:
```suggestion

CloudWatch Logs vous permet de centraliser les journaux de tous vos systèmes ainsi que les applications et les services AWSque vous utilisez au sein d’un seul service hautement évolutif. Vous pouvez ensuite facilement les consulter, y effectuer des recherches pour identifier des codes d’erreur spécifiques ou des modèles, les filtrer en fonction de champs spécifiques ou les archiver en toute sécurité à des fins d’analyse ultérieure.

> **Attention** Ce Plugin est susceptible de générer un volume importants de données lors des requêtes API.
> Il est indispensable d'utiliser les fonctionnalités de filtrage de celui-ci (```--group-name``` and ```--stream-name```) afin
> de limiter les résultats retournés par l'API.

## Dépendances du Connecteur de supervision
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md [hunk 2]
```diff
@@ -6,32 +6,58 @@ import Tabs from '@theme/Tabs';
...
 
 
-## Vue d'ensemble
-
-CloudWatch Logs vous permet de centraliser les journaux de tous vos systèmes ainsi que les 
-applications et les services AWSque vous utilisez au sein d’un seul service hautement évolutif. 
-Vous pouvez ensuite facilement les consulter, y effectuer des recherches pour identifier des 
-codes d’erreur spécifiques ou des modèles, les filtrer en fonction de champs spécifiques ou 
-les archiver en toute sécurité à des fins d’analyse ultérieure.
+CloudWatch Logs vous permet de centraliser les journaux de tous vos systèmes ainsi que les applications et les services AWSque vous utilisez au sein d’un seul service hautement évolutif. Vous pouvez ensuite facilement les consulter, y effectuer des recherches pour identifier des codes d’erreur spécifiques ou des modèles, les filtrer en fonction de champs spécifiques ou les archiver en toute sécurité à des fins d’analyse ultérieure.
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md:9]:
```suggestion
CloudWatch Logs vous permet de centraliser les journaux de tous vos systèmes ainsi que les 
applications et les services AWSque vous utilisez au sein d’un seul service hautement évolutif. 
Vous pouvez ensuite facilement les consulter, y effectuer des recherches pour identifier des 
codes d’erreur spécifiques ou des modèles, les filtrer en fonction de champs spécifiques ou 
les archiver en toute sécurité à des fins d’analyse ultérieure.
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md [hunk 3]
```diff
@@ -6,8 +6,6 @@ import Tabs from '@theme/Tabs';
 import TabItem from '@theme/TabItem';
 
 
-## Vue d'ensemble
-
 CloudWatch Logs vous permet de centraliser les journaux de tous vos systèmes ainsi que les 
 applications et les services AWSque vous utilisez au sein d’un seul service hautement évolutif.
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md:10]:
```suggestion
applications et les services AWS que vous utilisez au sein d’un seul service hautement évolutif. 
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md [hunk 4]
```diff
@@ -6,32 +6,62 @@ import Tabs from '@theme/Tabs';
...
 les archiver en toute sécurité à des fins d’analyse ultérieure.
 
-> **Attention** Ce Plugin est susceptible de générer un volume importants de données lors des requêtes API.
-> Il est indispensable d'utiliser les fonctionnalités de filtrage de celui-ci (```--group-name``` and ```--stream-name```) afin
+> **Attention**: Ce plugin est susceptible de générer un volume important de données lors des requêtes API.
+> Il est indispensable d'utiliser les fonctionnalités de filtrage de celui-ci (```--group-name``` et ```--stream-name```) afin
 > de limiter les résultats retournés par l'API.
 
-## Contenu du connecteur de supervision
+## Dépendances du Connecteur de supervision
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md:19]:
```suggestion
## Dépendances du connecteur de supervision
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md [hunk 5]
```diff
@@ -40,19 +70,14 @@ les archiver en toute sécurité à des fins d’analyse ultérieure.
 
 ### Privilèges AWS
 
-Voici la liste des droits nécessaires au travers des *access/secret keys* utilisées pour pouvoir implémenter 
-la supervision Amazon CloudWatch Logs: 
-
-| AWS Privilege                    | Description                                                                     |
-| :------------------------------- | :------------------------------------------------------------------------------ |
-| CloudWatchLogs:DescribeLogGroups | Returns information about LogGroups that meet the specified filter criteria.    |
+Configurez un compte de service (via une combinaison d'access et secret key) et affectez-lui les privilèges suivants :
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md:73]:
```suggestion
Configurez un compte de service (via une combinaison d'access key et secret key) et affectez-lui les privilèges suivants :
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md [hunk 6]
```diff
@@ -6,32 +6,59 @@ import Tabs from '@theme/Tabs';
...
 
-## Overview
-
 CloudWatch Logs enables you to centralize the logs from all of your systems, applications, 
 and AWS services that you use, in a single, highly scalable service. You can then easily view them, 
 search them for specific error codes or patterns, filter them based on specific fields, 
-or archive them securely for future analysis
+or archive them securely for future analysis.
 
 > This Plugin may generate very huge queries toward CloudWatch Logs API. Make sure to use filtering capability
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md:14]:
```suggestion
> This plugin may generate very huge queries toward the CloudWatch Logs API. Make sure you use the plugin's filtering capabilities
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md [hunk 7]
```diff
@@ -6,32 +6,59 @@ import Tabs from '@theme/Tabs';
...
 > with ```--group-name``` and ```--stream-name``` options.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md:15]:
```suggestion
> using the```--group-name``` and ```--stream-name``` options. 
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md [hunk 8]
```diff
@@ -6,32 +6,59 @@ import Tabs from '@theme/Tabs';
...
+The Monitoring Connector **Amazon CloudWatch Logs** brings a host template:
+
+* **Cloud-Aws-CloudWatchLogs-custom**
+
+The connector brings the following service templates (sorted by the host template they are attached to):
+
+<Tabs groupId="sync">
+<TabItem value="Cloud-Aws-CloudWatchLogs-custom" label="Cloud-Aws-CloudWatchLogs-custom">
+
+There is no service template linked to this host template.
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md:36]:
```suggestion
No service template is linked to this host template.
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md [hunk 9]
```diff
@@ -62,122 +87,294 @@ yum install perl-Paws
...
+	--aws-role-arn='' \
+	--proxyurl=''  \
+	--region='' \
+	--group-name='' \
+	--stream-name='' \
+	--start-time-since='' \
+	--unknown-status='' \
+	--warning-status='' \
+	--critical-status='' \
+	--verbose
```
Comment (cg-tw) [inline pp/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md:250]: the previous example was better

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md [hunk 10]
```diff
@@ -65,122 +90,294 @@ yum install perl-Paws
...
+	--aws-role-arn='' \
+	--proxyurl=''  \
+	--region='' \
+	--group-name='' \
+	--stream-name='' \
+	--start-time-since='' \
+	--unknown-status='' \
+	--warning-status='' \
+	--critical-status='' \
+	--verbose
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md:253]: same as EN

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md [hunk 11]
```diff
@@ -65,122 +90,294 @@ yum install perl-Paws
...
+	--plugin=cloud::aws::cloudwatchlogs::plugin \
+	--mode=get-logs \
+	--custommode='awscli' \
+	--aws-secret-key='' \
+	--aws-access-key='' \
+	--aws-role-arn='' \
+	--proxyurl=''  \
+	--region='' \
+	--group-name='' \
+	--stream-name='' \
```
Comment (cg-tw) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md:248]: ?

**Review by cg-tw — CHANGES_REQUESTED** [review]

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md [hunk 12]
```diff
@@ -65,122 +90,294 @@ yum install perl-Paws
...
-|             | DUMMYOUTPUT     | Host check output. Default is 'This is a dummy check'. Customize it with your own if needed |
+| Macro         | Description                                                                                                                | Valeur par défaut | Obligatoire |
+|:--------------|:---------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| AWSACCESSKEY  | Set AWS access key                                                                                                         |                   |             |
+| AWSCUSTOMMODE | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option | awscli            |             |
+| AWSREGION     | Set the region name (required)                                                                                             |                   |             |
+| AWSSECRETKEY  | Set AWS secret key                                                                                                         |                   |             |
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md:201]:
```suggestion
| AWSACCESSKEY  | Set AWS access key                                                                                                         |                   | X           |
| AWSCUSTOMMODE | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option | awscli            |             |
| AWSREGION     | Set the region name (required)                                                                                             |                   | X           |
| AWSSECRETKEY  | Set AWS secret key                                                                                                         |                   | X           |
```

File: i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md [hunk 13]
```diff
@@ -65,122 +90,294 @@ yum install perl-Paws
...
- --critical-status='%{message} =~ /region/i' \
- --verbose
+	--plugin=cloud::aws::cloudwatchlogs::plugin \
+	--mode=get-logs \
+	--custommode='awscli' \
+	--aws-secret-key='' \
+	--aws-access-key='' \
+	--aws-role-arn='' \
+	--proxyurl=''  \
+	--region='' \
```
Comment (lucie-dubrunfaut) [inline i18n/fr/docusaurus-plugin-content-docs-pp/current/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md:246]:
```suggestion
	--aws-secret-key='XXXX' \
	--aws-access-key='XXXX' \
	--aws-role-arn='' \
	--proxyurl=''  \
	--region='eu-west-1' \
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md [hunk 14]
```diff
@@ -62,122 +87,294 @@ yum install perl-Paws
...
- --critical-status='%{message} =~ /region/i' \
- --verbose
+	--plugin=cloud::aws::cloudwatchlogs::plugin \
+	--mode=get-logs \
+	--custommode='awscli' \
+	--aws-secret-key='' \
+	--aws-access-key='' \
+	--aws-role-arn='' \
+	--proxyurl=''  \
+	--region='' \
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md:243]:
```suggestion
	--aws-secret-key='XXXX' \
	--aws-access-key='XXXX' \
	--aws-role-arn='' \
	--proxyurl=''  \
	--region='eu-west-1' \
```

File: pp/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md [hunk 15]
```diff
@@ -62,122 +87,294 @@ yum install perl-Paws
...
+4. Fill in the macros you want. Some macros are mandatory.
+
+| Macro         | Description                                                                                                                | Default value     | Mandatory   |
+|:--------------|:---------------------------------------------------------------------------------------------------------------------------|:------------------|:-----------:|
+| AWSACCESSKEY  | Set AWS access key                                                                                                         |                   |             |
+| AWSCUSTOMMODE | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option | awscli            |             |
+| AWSREGION     | Set the region name (required)                                                                                             |                   |             |
+| AWSSECRETKEY  | Set AWS secret key                                                                                                         |                   |             |
```
Comment (lucie-dubrunfaut) [inline pp/integrations/plugin-packs/procedures/cloud-aws-cloudwatchlogs.md:200]:
```suggestion
| AWSACCESSKEY  | Set AWS access key                                                                                                         |                   | X           |
| AWSCUSTOMMODE | When a plugin offers several ways (CLI, library, etc.) to get information the desired one must be defined with this option | awscli            |             |
| AWSREGION     | Set the region name (required)                                                                                             |                   | X           |
| AWSSECRETKEY  | Set AWS secret key                                                                                                         |                   | X           |
```

**Review by cg-tw — APPROVED** [review]

**Pull request merged**